src/styles/Main.js

```diff
@@ -0,0 +1,53 @@
+import styled from 'styled-components';
+
+export const Container = styled.div`
+display: grid;
+justify-content: center;
+align-content: center;
+width: 400px;
+margin: 40px auto;
+grid-template-columns: repeat(4, 100px);
+grid-template-rows: minmax(120px, auto) repeat(5, 100px);
+box-shadow: 2px 2px 10px #333;
+border-radius: 10px;
+`;
```

Comment:
- [ ] Good job creating the initial component of your calculator, but as required, your design should match the given pattern as showed on the picture below 😄 

| Pattern Design | Your design | 
| :---: | :---: |
| ![Capture](https://user-images.githubusercontent.com/80458199/230612426-5748d554-5457-4674-800f-db4e4a971cce.PNG) |  ![Capture1](https://user-images.githubusercontent.com/80458199/230612531-01a684c4-15cc-455e-96cb-14026fa5d76b.PNG) | 

What do you think about fixing that ?